doc/nrf/ug_bt_mesh_vendor_model_dev_overview.rst

```diff
@@ -289,6 +289,11 @@ Defining model callbacks
 The Access API provides a set of callbacks that are called when certain events occur.
```

Comment:
The name of the file says vendor model development overview, and the actual page title says "Model development overview". I think it is better to be renamed as - "Vendor model development overview".

Reply:
It looks like there are a few other instances in the documentation where we refer to it just as "model", such as "Creating a new model", "Defining a model identifier", and "Adding the model to the node composition data". 

Does it make sense to change all these instances to "vendor model"? Does it sound misleading if we do not include "vendor"?

Reply:
I feel the title update is sufficient.  At other places, the context clarifies the meaning.

Reply:
Title is now updated to include "vendor".